infra/modules/s3/emails.tf

```diff
@@ -0,0 +1,30 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+resource "aws_s3_bucket_acl" "my_bucket" {
+  bucket = "octopus-email-bucket"
```

Comment:
Into the ticket description and also on aws, this bucket has a different name. I know TF it's going to create it with whatever name we give it but it might be confusing while working with this bucket